tests/session/cmd/test_show_config.py

```diff
@@ -81,22 +81,33 @@ def test_show_config_py_ver_impl_constants(tox_project: ToxProjectCreator) -> No
     assert outcome.out == f"[testenv:py]\npy_dot_ver = {py_ver}\npy_impl = {impl}\ndeps = {impl}{py_ver}\n"
 
 
-def test_show_config_exception(tox_project: ToxProjectCreator) -> None:
-    project = tox_project(
-        {
-            "tox.ini": """
-        [testenv:a]
-        base_python = missing-python
-        """,
-        },
-    )
-    outcome = project.run("c", "-e", "a", "-k", "env_site_packages_dir")
+@pytest.mark.parametrize(
+    "ini,key,expected_outcome",
+    [
+        (
+            """
+            [testenv:a]
+            base_python = missing-python
+            """,
+            "env_site_packages_dir",
+            "\nenv_site_packages_dir = # Exception: "
+            "RuntimeError(\"failed to find interpreter for Builtin discover of python_spec='missing-python'",
+        ),
+        (
+            """
+            [testenv:a]
+            install_command =
+            """,
+            "install_command",
+            "install_command = # Exception: " "ValueError(\"attempting to parse '' into a command failed\")",
+        ),
+    ],
+)
+def test_show_config_exception(tox_project: ToxProjectCreator, ini, key, expected_outcome) -> None:
```

Comment:
Only duplicating 3 lines so not worth the overhead.

Reply:
Seems reasonable, although I'm not going to push commits for that at the moment.  I'm going to eat a banana and play some chess.

Reply:
Yep, I'm going to take a longer pause from this pull request, and have no plans to make further edits.  You're welcome to make any further adjustments, and/or other people could pick up (or close) the task from here.

Reply:
After some holiday/downtime, I think I'm ready to attempt this again soon.  I'll plan to re-separate the test cases here, and may have to open a new pull request, since the original fork repository was deleted).